lib/egl/egl_main.cpp

```diff
@@ -437,27 +424,24 @@ EglMainThread::Handle EglMainThread::CreateWindow(EglWindow *caller, int w,
    }
 #endif //GLVIS_USE_EGL
 #ifdef GLVIS_USE_CGL
-   if (!out_hnd.isInitialized())
-   {
-      return out_hnd;
+   if (!out_hnd.isInitialized()) { return out_hnd; }
+   CGLError err = CGLSetCurrentContext(out_hnd.ctx.get());
+   if (err != kCGLNoError) { return out_hnd; }
+   // initialize GLEW before using any OpenGL functions
+   const auto status = caller->initGLEW(legacy_gl);
+   MFEM_VERIFY(status, "GLEW initialization failed!");
+   assert(glGetError() == GL_NO_ERROR);
```

Comment:
`glGetError()` could be checked in `initGLEW()`, so this (and the `#include`) would not be needed 🤔 .